Automattic/woocommerce-payments
# Migrate DisputeStatusChip comp to TS to improve code quality

Previously part of #6723 – split into separate PR to simplify review.

#### Changes proposed in this Pull Request









This PR migrates the `DisputeStatusChip` component to TypeScript to improve code quality.

No visual or functional changes are expected.

#### Testing instructions





* Ensure GH checks pass with no changes to snapshot tests.
* View a `DisputeStatusChip` to ensure it renders correctly – for example the Disputes List on **Payments → Disputes**
	<img width="182" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/eae3a1cf-c637-4674-be71-2dc9dd0c2797">

-------------------


**Post merge**
